src/seedu/addressbook/commands/FindCommand.java

```diff
@@ -48,13 +44,31 @@ public CommandResult execute() {
      */
     private List<ReadOnlyPerson> getPersonsWithNameContainingAnyKeyword(Set<String> keywords) {
         final List<ReadOnlyPerson> matchedPersons = new ArrayList<>();
+        Set<String> lowerCaseKeywords = changeToLowerCase(keywords);
         for (ReadOnlyPerson person : addressBook.getAllPersons()) {
             final Set<String> wordsInName = new HashSet<>(person.getName().getWordsInName());
-            if (!Collections.disjoint(wordsInName, keywords)) {
+            Set<String> lowerCaseWordsInName = changeToLowerCase(wordsInName);
+            if (!Collections.disjoint(lowerCaseWordsInName, lowerCaseKeywords)) {
                 matchedPersons.add(person);
             }
         }
         return matchedPersons;
     }
 
+    /**
+     * Changes all words to lower case to remove issue of case sensitivity when comparing strings
```

Comment:
IMO, since this is method has no side-effects i.e. functional, there is no need to mention 'to remove issue of case sensitivity when comparing strings'